build/webpack/webpack.titleBar.js

```diff
@@ -107,6 +107,10 @@ module.exports = {
         path: path.resolve(__dirname, '../../dist/')
     },
     resolve: {
```

Comment:
This wasn't necessary before this PR beacuse the react-controls provided their own react and react-dom version. the ui does not.

Reply:
👍